lkmpg.tex

```diff
@@ -412,31 +412,35 @@ \subsection{Passing Command Line Arguments to a Module}
 I would recommend playing around with this code:
 \begin{code}
 $ sudo insmod hello-5.ko mystring="bebop" myintarray=-1
-myshort is a short integer: 1
-myint is an integer: 420
-mylong is a long integer: 9999
-mystring is a string: bebop
-myintarray[0] = -1
-myintarray[1] = 420
-got 1 arguments for myintarray.
+$ dmesg | tail -7
+[191207.935945] myshort is a short integer: 1
```

Comment:
Do not leave the kernel’s timestamps here, because it varies over time.

Reply:
Time tag removed.

Reply:
$ dmesg -t | tail -f would easily get kernel output without timestamps and data appended data as file grows. :D